### DISCUSSION TITLE
Green square around red motion marker in debug view

### DISCUSSION THREAD

**Discussion by Matzebhv** [blakeblackshear/frigate#6341] [Ask A Question] [answered]
Hi Folks,

on 2 of my cams around the red motion marker i have a bigger green square. I have read elsewhere that this is the area send to the object detection mechanism. But why do only 2 cams shows this green square?
Regards
Matthias

**Comment by NickM-27** [comment, marked as answer]
You can configure which boxes are shown in the debug view with the debug control panel below the debug view

**Reply by Matzebhv** [reply]
Regions where unchecked, Sorry for the dumb question and thx for the quick answer.
